### PR TITLE
ENH: Implement BIDS filters file and drop legacy BIDS querying

### PR DESCRIPTION
**Pull request by oesteban**
Requires: nipreps/niworkflows#833.

Resolves: #760 .

### CODE REVIEW

File: mriqc/cli/parser.py [hunk 1]
```diff
@@ -520,20 +520,29 @@ def parse_args(args=None, namespace=None):
...
-            config.execution.layout, **bids_filters
-        ).items()
-        if files
+    lc_modalities = [mod.lower() for mod in config.execution.modalities]
+
+    bids_filters = (
+        {mod: {"run": config.execution.run_id} for mod in lc_modalities}
+        if config.execution.run_id
+        else {}
+    )
```
Comment (effigies) [inline mriqc/cli/parser.py:529]: Do you want to go ahead and support bids filter files? https://github.com/nipreps/mriqc/issues/760

```suggestion
    bids_filters = <get base bids_filters from args>
    if config.execution.run_id:
        for mod in lc_modalities:
            bids_filters.setdefault(mod, {})["run"] = config.execution.run_id}
```

Comment (oesteban) [reply mriqc/cli/parser.py]: Good idea.

**Comment by Remi-Gau** [comment]
@oesteban 
random question: quick look at the code makes me think this only works at the individual level, no?
if one wanted to use the bids filter to make group summary plot of only a subset of images, this filtering would not work, right?

**Comment by oesteban** [comment]
> quick look at the code makes me think this only works at the individual level, no?

Correct

> if one wanted to use the bids filter to make group summary plot of only a subset of images, this filtering would not work, right?

This would be a good idea, but as you expect, it will not work ATM. However, it is definitely a good idea for any takers.

**Pull request merged**